docs/chapters/architecture/SOTA-system.rst

```diff
@@ -0,0 +1,765 @@
+"core-image-full-cmdline" image and to platforms such as the wandboard or
+beaglebone which aren't supported by PELUX.
+
+meta-swupdate has been integrated to the default PELUX manifests, to avoid code
+duplication. However, it has been decided that the meta-swupdate-boards
+contained too much superfluous code and the few relevant parts of that layer
+have been directly integrated in meta-pelux.
+
+Quality requirements
+--------------------
+
+The quality requirements are documented in the Integration and goals part of
+this document.
+
+Risks and technical debts
+-------------------------
+
+The SOTA System is a critical component in an automotive system and presents a
+lot of risks. The architecture presented in this document mitigates those risks
+by taking into account early in the design process the need for atomic updates
+and artifacts signature.
+
+Technical debts are minimized with the chosen architecture because SWUpdate and
+Hawkbit are widely used in the industry and supported by stable companies and
+foundations such as Bosch, Eclipse or DENX. We can expect those components to be
+supported on the long term. Those components have also been chosen for their
+flexibility that should effectively adapt to new use cases.
+
+Glossary
+--------
+
+See the :ref:`arch-glossary` page for explanations of important terms.
+
+Appendix: Getting started with Hawkbit and SWUpdate
+---------------------------------------------------
+
+Due to a lack of clear instructions available on the internet, this appendix
+details the necessary steps to setup a local installation of Hawkbit and
+interface it with a PELUX system. This should be enough to set up a local
+development environment but extra steps would be needed for a real updates
+deployment context.
+
+Context
+^^^^^^^
+
+For the rest of the tutorial, we will assume you have two machines connected
+on an IP network:
+
+- A development machine running a standard Linux distribution. We will assume
+  that this machine has the *192.168.3.11* IP address. This machine must have
+  Java installed.
+- A raspberrypi3 running PELUX. 
+
+Compiling SWUpdate artifacts
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can generate update artifacts of PELUX from your Yocto build directory
+using bitbake:
+
+.. code-block:: bash
+
+    $ bitbake core-image-pelux-minimal-update
+
+The resulting file can then be found at:
+*build/tmp/deploy/images/raspberrypi3/core-image-pelux-minimal-update-raspberrypi3.swu*
+
+Hawkbit installation
+^^^^^^^^^^^^^^^^^^^^
+
+We will fetch Hawkbit from its GitHub repository.
+
+.. code-block:: bash
+
+    $ git clone https://github.com/eclipse/hawkbit
+    $ cd hawkbit
+
+Recent versions of Hawkbit aren't yet supported by SWUpdate so we need to
+manually select a slightly older version of Hawkbit.
+
+.. code-block:: bash
+
+    $ git checkout 0.2.0M4
+
+We can now compile Hawkbit using Maven.
+
+.. code-block:: bash
+
+    $ mvn clean install
```

Comment:
Is there a dependencies chapter describing the dependencies of the server? Couldn't find one :)

Reply:
I added information about Java and Maven requirements in the "Context" subsection of this appendix. ;)